File: spec/code_samples/Python_requests/alerts@{alert_type}/delete.py
import requests

url = "https://www.directfreight.com/api/alerts/%7Balert_type%7D"

querystring = {"alert_id":"SOME_ARRAY_VALUE"}

headers = {'end-user-token': 'SOME_STRING_VALUE'}

response = requests.request("DELETE", url, headers=headers, params=querystring)

print(response.text)

File: spec/code_samples/Php_http2/alerts@{alert_type}/delete.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('https://www.directfreight.com/api/alerts/%7Balert_type%7D');
$request->setRequestMethod('DELETE');
$request->setQuery(new http\QueryString(array(
  'alert_id' => 'SOME_ARRAY_VALUE'
)));

$request->setHeaders(array(
  'end-user-token' => 'SOME_STRING_VALUE'
));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();